ios/RCTWebRTC/WebRTCModule+Daily.m

```diff
@@ -144,58 +145,22 @@ - (void)audioSession:(RTCAudioSession *)audioSession didSetActive:(BOOL)active {
   // Apply the chosen audio mode right away if the audio session is already
   // active. Otherwise, it will be applied when the session becomes active.
   RTCAudioSession *audioSession = RTCAudioSession.sharedInstance;
+  NSLog(@"[Daily] setDailyAudioMode: %@", audioMode);
```

Comment:
Did you meant to keep these debug logs in here?

Reply:
I was think in keep those debug logs, what do you think ?